pkgs/development/python-modules/ploomber-extension/default.nix

```diff
@@ -10,14 +9,13 @@
   ploomber-core,
   pytestCheckHook,
   pytest-jupyter,
+  stdenv,
```

Comment:
Nit: `stdenv` should be at the top, right after `lib`.

Reply:
Got it. Thank you.